src/flows_dst.rs

```diff
@@ -669,6 +678,78 @@ impl<'a> ProcessMerge<'a> {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct ProcessSplit<'a>(pub &'a mut MemberState);
+
+impl<'a> ProcessSplit<'a> {
+    pub fn start_event_loop(&mut self) {
+        self.mut_routine_state().is_active = true;
+        self.vote_for_section_split();
+    }
+
+    fn exit_event_loop(&mut self) {
+        self.mut_routine_state().is_active = false;
+        self.0
+            .as_start_merge_split_and_change_elders()
+            .transition_exit_process_split()
+    }
+
+    pub fn try_next(&mut self, event: WaitedEvent) -> TryResult {
+        match event {
+            WaitedEvent::ParsecConsensus(vote) => self.try_consensus(&vote),
+            _ => TryResult::Unhandled,
```

Comment:
as for merge, can you list them all. (had to change them a few times, makes compiler helpful.